update-dependency-pinning.sh

```diff
@@ -3,5 +3,5 @@
 WITNESS=app/witness.gradle
 
 echo "" > $WITNESS
-./gradlew -q calculateChecksums | grep -Ev "^(Skipping|Verifying)" | grep -Ev "files-2.1:|:build-tools:core-lambda-stubs.jar:|:platforms:android.jar:|-linux.jar:" > $WITNESS
+./gradlew -q calculateChecksums | grep -Ev "^(Skipping|Verifying)" | grep -Ev "files-2.1:|caches:transforms-3:|:build-tools:core-lambda-stubs.jar:|:platforms:android.jar:|-linux.jar:" > $WITNESS
```

Comment:
Had to filter these out as they seemed to be again some intermediate files. We should eventually try to leave Gradle witness behind...

Reply:
Last time I tried that built-in gradle mechanism it worked even worse.

Reply:
Really? I tried it out just yesterday for Transportr, and it seemed to work quite well and straightforward.

Reply:
Maybe things got better in gradle 7?